sys/fido2/doc.txt

```diff
@@ -17,6 +17,12 @@
  *
  * @warning The FIDO2 implementation currently stores private keys in plain text inside flash memory.
  *
+ * @warning This implementation does not support firmware updates if riotboot is used in
```

Comment:
From my understanding this could be:

> @warning This implementation only persists FIDO CTAP data across reboots and when unpowered; any firmware update loses the data. This applies both to firmware updates through bootloaders and to firmware updates through external programmers.